backend/src/handlers/user_handler.go

```diff
@@ -37,8 +37,10 @@ func (srv *Server) HandleIndexUsers(w http.ResponseWriter, r *http.Request) {
 		srv.HandleGetUnmappedUsers(w, r, providerId)
 		return
 	}
+	order := r.URL.Query().Get("order")
+	search := strings.ToLower(r.URL.Query().Get("search"))
```

Comment:
Actually I forgot this when I pushed the changes:

Can we add a 
```go
search = strings.TrimSpace(search)
```
right below this? without it, it will neglect to return some results when someone pushes the spacebar in an empty search bar.

Reply:
yes.  I applied the same to user_activity_handler as well.